experimental/benchmark/src/main.rs

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+mod proto {
+    tonic::include_proto!("oak.examples.trusted_database");
+}
+
+mod backend;
+mod database;
+mod runtime;
+
+use crate::{backend::TestBackend, database::Database};
+use log::info;
+use runtime::TestRuntime;
+use std::time::Instant;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Clone)]
+#[structopt(about = "Trusted Database Client")]
+pub struct Opt {
+    #[structopt(long, help = "Database size (number of items)")]
+    database_size: usize,
+}
+
+#[tokio::main(core_threads = 4)]
+async fn main() -> anyhow::Result<()> {
```

Comment:
Would be nice to have a high-level documentation here explaining the experiment; in particular mentioning the difference between the `Backend` and the `TestRuntime`.